hasura/metadata/tables.yaml

```diff
@@ -281,16 +281,20 @@
           email:
             _eq: x-hasura-user-email
         columns:
-          - email
           - botNotifications
+          - current_login
+          - email
+          - last_login
```

Comment:
`"reason": "Inconsistent object: in table \"User\": in permission for role \"world\": column \"last_login\" does not exist"`

In Hasura the column is called "last_seen" and also, wordl has not select permission on last_seen and current_login
![Capture d’écran de 2022-11-29 10-42-56](https://user-images.githubusercontent.com/85115963/204494701-2115a9f0-9b6a-45d6-81c9-80f4e660f558.png)

Reply:
It's weird, did you apply all migrations ? Because on my side, I've the right name and permissions

Reply:
Yes i've done : 
`npm run hasura migrate apply && npm run hasura metadata apply`

But the message above was displayed during the metadata apply. 

So I tried to stop and rm my docker volumes then apply the seeds to get the column. I don't know why. 

Now it works, but I had to reset all my local datas (wtf ?)